test/native_objc_test/swift_class_test.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
```

Comment:
Are Modules also an ObjectiveC concept? If so, we should add an ObjectiveC test next to the Swift test.

Reply:
No, I can't figure out how to write ObjC code that gets compiled with a module prefix. It seems to be a Swift only thing.